packages/foundations-buttons/src/buttons.pcss

```diff
@@ -4,89 +4,106 @@
 
 .coop-btn {
   display: inline-block;
-  padding: var(--spacing-base--1-4) calc(var(--spacing-base--1-2) * 1.25);
+  padding: calc(var(--spacing-8) * 1.25) calc(var(--spacing-16) * 1.25);
   color: var(--color-white);
-  background: var(--color-button-dark);
-  font-size: var(--type-base--body);
+  background: var(--color-button-blue);
+  font-size: var(--type-body-s);
   font-family: var(--font-family);
   font-weight: 500;
   line-height: 1.2;
   text-align: center;
   border: 0;
-  border-radius: 7px;
-  transition: all 0.15s linear;
+  border-radius: var(--ui-border-radius);
+  transition: var(--ui-transition-hover);
   outline: none;
   text-decoration: none;
   cursor: pointer;
 
   @media (--mq-medium) {
-    font-size: var(--type-medium--body);
-    padding: var(--spacing-medium--1-4) calc(var(--spacing-medium--1-2) * 1.25);
+    font-size: var(--type-body-l);
+    padding: var(--spacing-16) calc(var(--spacing-32) * 1.25);
   }
 }
 
 .coop-btn:hover,
 .coop-btn:focus {
-  background: var(--color-button-dark--hover);
+  background: var(--color-button-blue-hover);
   color: var(--color-white);
   border: 0;
+  text-decoration: underline;
 }
 
 .coop-btn:focus {
   transition: none;
-  outline: 2px solid var(--color-link--focus);
+  outline: 2px solid var(--color-link-focus);
```

Comment:
This colour is the link text focus colour -- the double-hyphen was previously used for the focus outline, but this seems a bit of a misnomer now. Should this variable be called `--colour-focus-outline` maybe?